src/EditorFeatures/Text/Shared/Extensions/ITextSnapshotExtensions.cs

```diff
@@ -120,12 +120,12 @@ public static NormalizedSnapshotSpanCollection GetSnapshotSpanCollection(this IT
             return new NormalizedSnapshotSpanCollection(snapshot.GetFullSpan());
         }
 
-        public static void GetLineAndColumn(this ITextSnapshot snapshot, int position, out int lineNumber, out int columnIndex)
+        public static void GetLineAndCharacter(this ITextSnapshot snapshot, int position, out int lineNumber, out int characterIndex)
```

Comment:
can this change be extracted into its own PR?  it seems unnecessary for the overall change (esp. as this is going into dev16).  Was there a reason this needed to part of this PR?

Reply:
➡️ Already extracted to #30865